ag/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: Mihail Konev <k.mvc@ya.ru>
```

Comment:
IMO this folder should be named `ag-git`. 
There should be another folder `ag` which installs stable release version.

The same applies to `cgvg`.